src/main/resources/db/changelog/db.changelog-references.xml

```diff
@@ -125,7 +125,7 @@
         <createTable tableName="case_tags">
             <column name="CASE_ID" type="BIGINT(19)">
                 <constraints nullable="false"
-                    foreignKeyName="fk_caseTag_case" references="suit(id)"/>
+                    foreignKeyName="fk_caseTag_case" references="case(id)"/>
```

Comment:
@pomkine Can we change existing schema descriptor, or must create new changeset to fix the bug?

Reply:
@hdghg  I think we can edit the existing schema because we are not release yet. We can drop the db easily :)